app/controllers/api/api_controller.rb

```diff
@@ -13,6 +13,7 @@ class ApiController < ActionController::API
     rescue_from ActiveRecord::RecordNotUnique, with: :bad_request_response
     rescue_from ActiveRecord::RecordInvalid, with: :invalid_transition
     rescue_from Api::Errors::InvalidTransitionError, with: :invalid_transition
+    rescue_from Api::Errors::NPQApplicationAlreadyAcceptedError, with: :invalid_transition
```

Comment:
It's probably because it ends up as a 422, but I normally think of state changes when I see "invalid_transition"

Reply:
we can have duplicate method with the same name